examples/cx16/k_cbm_file_test.c

```diff
@@ -0,0 +1,282 @@
+// llvm-mos-sdk cx16 kernel file access tests
+//
+// vim: set et ts=4 sw=4
```

Comment:
Add licence infor and author

Reply:
Instead of the vim comment, rely on `.clang-format` (see above comment) or perhaps `.editorconfig` (https://editorconfig.org).

Reply:
None of the other examples I peeked at have license or author.  I prefer the options per file in easy to edit or see comment (vim: stuff).  clang-format is different, I wish you had a .clang-format (like the lllvm one) in the llvm-mos-sdk tree (and it would automatically get used).

Reply:
> None of the other examples I peeked at have license or author.

You're right there's no file-level license info elsewhere in the SDK and I guess the LICENSE file at top-level suffices. The llvm project often adds a header to source files, though.

Reply:
File-level licensing information is a good practice to minimize ambiguity.

Reply:
I would be fine with adding some info to the cx16 examples (just following other examples).  Maybe a few other ones can get updated and I will copy the "style" and language for the cx16 ones.

Reply:
There's already a requirement to include a copy of the Apache license with any distribution of the sources; per-file license just provide a backup if this isn't done, since it's separately a license violation to remove them, and that takes an additional action, rather than just say forgetting to include LICENSE.

Accordingly, I haven't been in the habit for requiring Apache license headers outside of files where there already license headers; there the callthrough to the LICENSE file should always be made explicit, since otherwise the existing headers would seem to supercede LICENSE. The llvm-mos repo is also a special case, since it intended to look as much as possible like the surrounding files.

That being said, I'm not opposed to having license headers, or further discussion along this line. If folks would for instance feel put out by having these files appear somewhere else without attribution, that would be a good reason to add headers. I just don't know of a compelling reason on my end.

> I wish you had a .clang-format (like the lllvm one) in the llvm-mos-sdk tree (and it would automatically get used).
I'll second this; I'd prefer to see a copy of the `.clang-format` in llvm-mos added to this repo. We should do this in a separate step with an SDK-wide refactoring though; it cuts the git history, and we should only do that once. Accordingly, this should be out of scope for this PR.